starters/vue3-xstate-css/src/views/HomeView.vue

```diff
@@ -0,0 +1,35 @@
+<script setup lang="ts"></script>
+
+<template>
+	<main>
+		<div class="home__title">
+			<h1>Vue 3 + XState + CSS</h1>
+		</div>
+		<div class="home__links">
```

Comment:
small accessibilty suggestion, make this a <nav> instead than a div

Reply:
Looks like something went weird with the text - what should this be? A section maybe?

Reply:
A nav